knotx-example/knotx-example-monolith/src/test/java/com/cognifide/knotx/monolith/util/FileReader.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Knot.x - Sample App with Mock service
```

Comment:
File reader utility already exists and is simpler. Check tests in template engine module